prometheus/filter.go

```diff
@@ -2,10 +2,11 @@ package prometheus
 
 import (
 	"sort"
+	"strings"
 
-	"github.com/golang/protobuf/proto"
 	"github.com/prometheus/client_golang/prometheus"
 	dto "github.com/prometheus/client_model/go"
+	"google.golang.org/protobuf/proto"
```

Comment:
What caused the import change for the different proto library?

Reply:
The `github.com/golang/protobuf` package is fully deprecated, which caused problems with staticcheck

Reply:
Or at least the `proto` package within it, not sure which

Reply:
Can you link to some docs about compatibility between "google.golang.org/protobuf/proto" and "github.com/golang/protobuf", for posterity?